SwiftLeeds/Views/Common/CommonCompo.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  CommonCompo.swift
+//  SwiftLeeds
+//
+//  Created by LUCKY AGARWAL on 23/07/22.
+//
+
+import Foundation
+import SwiftUI
+
+class CommonCompo {
+    
+    static func sectionHeader(text: String) -> some View {
```

Comment:
Yeah, you could just lift this out into a 'struct SectionHeader: View' 

The idea of a 'CommonCompo' is great, but these files end up huge and it gets really hard to work with

Reply:
Alrighty. Will do that